Makefile.core.mk

```diff
@@ -61,6 +62,8 @@ kube_api_packages = $(subst $(space),$(empty), \
 	$(kube_api_base_package)/security/v1beta1, \
 	$(kube_api_base_package)/telemetry/v1alpha1 \
 	)
+# this is needed to properly generate ssa functions
+kube_api_applyconfiguration_packages = $(kube_api_packages),k8s.io/apimachinery/pkg/apis/meta/v1
```

Comment:
It's necessary to have `k8s.io/apimachinery/pkg/apis/meta/v1` to have the proper method signature for `WithOwnerReferences` generated. If this is not specified then the generated code cannot be compiled (wrong import is used). This is the same as done in https://github.com/kubernetes/kubernetes/pull/99012